extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -880,6 +884,13 @@ export const popupMessageListener = (request: Request) => {
 
     const currentIsExperimentalModeEnabled = getIsExperimentalModeEnabled();
 
+    await dataStorage.setItem({
+      [DATA_SHARING_ID]: isDataSharingAllowed,
+      [IS_VALIDATING_MEMO_ID]: isMemoValidationEnabled,
+      [IS_VALIDATING_SAFETY_ID]: isSafetyValidationEnabled,
+      [IS_VALIDATING_SAFE_ASSETS_ID]: isValidatingSafeAssetsEnabled,
+    });
```

Comment:
this is a preview of what the new "save" API will look like.

For now, recording new settings in both browser storage and local storage. We'll remove the latter as we continue ton refactor

Reply:
love how it can be done w/ one method call 💪 

can you add a todo line here since some of these lines are technically "repeat" for a short amount of time, so easy to remember later?